internal/lua/module_test.go

```diff
@@ -459,6 +459,81 @@ func Example_api() {
 	//Result:  success
 }
 
+// Run simple test with topics API functions
+func Example_topics_api() {
+	ctx := context.Background()
+	args := map[string][]string{}
+	files := map[string][]byte{
+		"main.lua": []byte(`
+			-- patch local environment because it's not initialized yet
```

Comment:
I think it would be better to have these Lua files in separate files in testdata folder, that would simplify reading then in IDE/maybe running them locally/etc.

Reply:
good idea but it's out of scope this PR